lib/kdl/test/src/collection_utils_test.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <gtest/gtest.h>
 
+#include "test_utils.h"
```

Comment:
I try to group imports and would suggest an empty line after the `test_utils` include.